core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileMain.scala

```diff
@@ -62,13 +63,27 @@ object ProfileMain extends Logging {
       eventLogFsFiltered
     }
 
-    if (filteredLogs.isEmpty) {
-      logWarning("No event logs to process after checking paths, exiting!")
+    if (filteredLogs.isEmpty && driverLog.isEmpty) {
+      logWarning("No event logs to process after checking paths and no driver log " +
+        "to process, exiting!")
       return (0, filteredLogs.size)
     }
 
+    // Check only one eventlog is provided when driver log is passed for non-databricks eventlog
+    // For databricks eventlog, eventlogs are stored in a directory with rolled multiple
+    // eventlog files
+    if(driverLog.nonEmpty && filteredLogs.size > 1 &&
+      !filteredLogs.forall(_.isInstanceOf[DatabricksEventLog])) {
```

Comment:
I'm confused by this, filteredLogs at this point already knows the event log type right?  ie your check for DatabricksEventLog, so if filteredLogs.size is > 1 doesn't that mean there are multiple databricks event log directories?

Reply:
You are right. We just need to know if there are multiple event logs in filteredLogs at this point irrespective of whether it's databricks eventlog or non-databricks eventlog.